be/src/storage/lake/versioned_tablet.h

```diff
@@ -82,6 +82,10 @@ class VersionedTablet {
     StatusOr<std::unique_ptr<TabletReader>> new_reader(Schema schema, bool could_split, bool could_split_physically,
                                                        const std::vector<BaseRowsetSharedPtr>& base_rowsets);
 
+    StatusOr<std::unique_ptr<TabletReader>> new_reader(Schema schema, bool could_split, bool could_split_physically,
```

Comment:
consider merge the interface with the previous one by setting the last parameter default to empty shared_ptr?